frontend/src/views/Gallery.vue

```diff
@@ -216,23 +213,14 @@ onMounted(async () => {
   } else {
     romsStore.setCurrentPlatform(routePlatform);
   }
-
-  // If platform is different, reset store and fetch roms
-  if (storedPlatformId != routePlatformId && !noPlatformError.value) {
```

Comment:
don't need this check anymore?

Reply:
Yeah I optimized the gallery reset and new platform detection handling so this is not needed anymore. Actually it will break the gallery if we readded it as how it's the code now.